src/plugins/discover/public/application/main/components/layout/discover_layout.tsx

```diff
@@ -236,7 +241,13 @@ export function DiscoverLayout({
           spaces={spaces}
           history={history}
         />
-        <h1 id="savedSearchTitle" className="euiScreenReaderOnly">
+        <h1
```

Comment:
we could think of moving the `<h1>` to L223, so it's the first after the TopNavMemoized. Because when using tab currently the side bar field search gets focus. I think starting with the data view picker would be a better fit.

Reply:
@kertal Great call. If the H1 can be moved to this new position, it would be a good reference point for future applications. We could use this as guidance to say "Pages and applications should have an H1 just after the memoized top navigation" as common guidance.

I'm attaching a screenshot I took from my local machine this morning with annotations to make sure I'm understanding the new location as you've described it.

---
<img width="898" alt="Screen Shot 2022-06-09 at 8 47 53 AM" src="https://user-images.githubusercontent.com/934879/172864537-0e7cbb78-f34f-4db4-9fe7-22ff19bb9730.png">

Reply:
yes, that's what I meant and also it work when doing it that way (short local test with Davis PR)